src/TestData/Framework/Provider/RandomAuthor.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace GiveTestData\TestData\Framework\Provider;
+
+/**
+ * Returns a random Author ID from the users table.
+ */
+class RandomAuthor extends RandomProvider {
```

Comment:
`RandomAuthor` should probably have some restrictions on which users can be returned, such as a role or capability check.

Reply:
Ref 831e9f39